provider/terraform_object_library.rb

```diff
@@ -89,7 +89,7 @@ def copy_common_files(output_folder, _version_name)
                        ['google/self_link_helpers.go',
                         'third_party/terraform/utils/self_link_helpers.go'],
                        ['google/utils.go',
```

Comment:
Ah, my bad- turns out we need to change lists on this file as well. Sorry for the round trips!

If you don't mind moving the entry to https://github.com/GoogleCloudPlatform/magic-modules/pull/1624/files#diff-663ce2cfe35833f7fb0fb01e2813f419R45